docs/api/covidcast-signals/dsew-cpr.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Data Strategy and Execution Workgroup Community Profile Report
+parent: Data Sources and Signals
+grand_parent: COVIDcast Epidata API
+---
+
+# Data Strategy and Execution Workgroup Community Profile Report (CPR)
+{: .no_toc}
+
+* **Source name:** `dsew-cpr`
+* **Earliest issue available:** 2022-01-28
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** county, msa, state, hhs, nation (see [geography coding docs](../covidcast_geography.md))
+* **Time type:** day (see [date format docs](../covidcast_times.md))
+* **License:** [Open Data Commons Open Database License (ODbL)](https://opendatacommons.org/licenses/odbl/1-0/)
```

Comment:
How do we know it's under the ODbL? I don't see a license listed on the healthdata.gov page for it, and by default works of the United States Government are in the public domain (not subject to copyright).

Reply:
I set the license here to match how we document [our HHS data](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/hhs.html). Both datasets are listed as "Public Domain U.S. Government" on healthdata.gov.

@krivard for further guidance.

Reply:
Weird. 

We added HHS in December 2020. At the time, it was listed as licensed under ODbL, probably because API access is handled through Socrata.

[The first snapshot from the Wayback Machine is from March 2021](https://web.archive.org/web/20200915000000*/https://healthdata.gov/Hospital/COVID-19-Reported-Patient-Impact-and-Hospital-Capa/g62h-syeh), where it is listed as licensed Public Domain: US Government.

Use "Public Domain: US Government" here. I'll open an issue to update the licensing information for HHS.